style.css

```diff
@@ -1,13 +1,16 @@
-@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@600&display=swap');
-
+/* Import  font from google fonts*/
```

Comment:
I import font from google fonts. The fonts I choice is Source sans pro and Open sans